.github/workflows/wasm.yaml

```diff
@@ -0,0 +1,27 @@
+---
+name: "Build WASM"
+on: # yamllint disable-line rule:truthy
+  push:
+    branches:
+      - "main"
+    paths:
+      - "pkg/development/wasm/**"
```

Comment:
This path isn't sufficient, since if we make a development package change, we need to rebuild as well. We likely need to have any paths, except the `dist` directory